wtpython/display.py

```diff
@@ -146,17 +147,16 @@ async def action_open_browser(self) -> None:
 
     async def action_open_google(self) -> None:
         """Open the browser with google search results"""
-        if self.data["error_message"]:
-            params = {'q': f"python {self.data['error_message']}"}
-            url = 'https://www.google.com/search?' + urlencode(params)
-            webbrowser.open(url)
+        params = {'q': f"python {''.join(traceback.format_exception_only(type(RAISED_EXC), RAISED_EXC)).strip()}"}
+        url = 'https://www.google.com/search?' + urlencode(params)
+        webbrowser.open(url)
 
     async def on_startup(self, event: events.Startup) -> None:
         """App layout"""
-        self.title = f"{APP_NAME} | {PARSED_TB['error_message']}"
+        self.title = f"{APP_NAME} | {''.join(traceback.format_exception_only(type(RAISED_EXC), RAISED_EXC)).strip()}"
```

Comment:
...It seems we're doing this a lot. Maybe just compute this once when we get the exception and then just monkey-patch the `Exception` object with this message?
```py
exc.display_message = ''.join(traceback.format_exception_only(type(exc), exc)).strip()
```

Reply:
I recognize that i have this once in main and twice in display, but I'm not sure it's quite worth creating a new function. I'm gonna leave it like it is for now, but we can always refactor it a bit later.